src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,73 @@
+package wolox.training.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.exceptions.BookIdMismatchException;
+import wolox.training.repositories.BookRepository;
+import wolox.training.models.Book;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/books")
+public class BookController {
+
+    @Autowired
+    private BookRepository bookRepository;
+
+    @GetMapping
+    public Iterable findAll() {
+        return bookRepository.findAll();
+    }
+
+    @GetMapping("/title/{bookTitle}")
+    public List findByTitle(@PathVariable String bookTitle) {
+        try {
+            return bookRepository.findByTitle(bookTitle);
```

Comment:
You can move this code to a service, in this case "BookService". It's a good practice to call services from the controller, call the repositories in the services. Check this post https://medium.com/karibu-blog/c%C3%B3mo-aplicar-jpa-en-spring-boot-e45ac5bee3b4